src/components/header/Header.tsx

```diff
@@ -7,7 +7,7 @@ import { defaultFlexContainer, desktopOnly, mobileOnly } from '../../styles/vads
 import { onHashAnchorClick } from '../../utils/clickHandlers';
 import Banner from '../Banner';
 import VeteransCrisisLine from '../crisisLine/VeteransCrisisLine';
-import NavBar from '../NavBar';
+import NavBar from '../navBar/NavBar';
```

Comment:
This needs to be updated as it doesn't match the case of the path.
```suggestion
import NavBar from '../NavBar/NavBar';
```

Reply:
Good catch, somehow the folder name didn't properly update to `navBar`